CMakeLists.txt

```diff
@@ -203,6 +203,8 @@ set(XEUS_CPP_SRC
     src/xoptions.cpp
     src/xparser.cpp
     src/xutils.cpp
+    src/xmagics/xassist.hpp
```

Comment:
We should make these conditional to #ifdef __EMSCRIPTEN__ in cmake. Also
We should split the header file and the source file in the previous list.